docs/index.md

```diff
@@ -114,6 +196,16 @@ class YourCoreLibInstance(object):
     def get() -> YourCoreLib:
         return YourCoreLibInstance._app_instance        
 ```
+#### Code Explained:
```

Comment:
the example above it have indentation problem. 
<img width="646" alt="image" src="https://github.com/shay-te/core-lib/assets/5166849/b07ae528-5dbb-4421-852c-c5e7f39e999b">